components/atom/spinner/README.md

```diff
@@ -0,0 +1,27 @@
+# AtomSpinner
+
+> An animated loop used for giving users feedback while the content of a page or section is being loaded.
+
+<!-- ![](./assets/preview.png) -->
+
+## Installation
+
+```sh
+$ npm install @schibstedspain/sui-atom-spinner --save
+```
+
+## Usage
+
+### Basic usage
+```js
+import AtomSpinner, {AtomSpinnerTypes} from '@schibstedspain/sui-atom-spinner'
+
+return (
+  <AtomSpinner on type={AtomSpinnerTypes.FULL}>
```

Comment:
Can you describe somewhere what the options FULL and SECTION mean ?

Reply:
Will add it as prop type description